polygeist/tools/cgeist/driver.cc

```diff
@@ -667,8 +667,6 @@ static LogicalResult finalize(mlir::MLIRContext &Ctx,
       // PM3.addPass(mlir::createLowerFuncToLLVMPass(options));
       PM3.addPass(polygeist::createLegalizeForSPIRVPass());
 
-      // Needed because SYCLMethodOps lowering might introduce redundant
-      // operations.
       PM3.addPass(mlir::createCSEPass());
```

Comment:
CSE pass can be removed as well?

Reply:
I'd keep it, as our lowering might be introducing redundant operations anyway

Reply:
ok